def example3(x, y, z):
	a = x + y
	return x + y

def example4(x, y, z):
	return x 



# para el  caso adicional example3 demuestra que una variable asignada dentro de la funcion no utilizada,
# no levanta el error, ya que solo se levanta el error de z no utilizada, y no de a, ya que a no
# es un argumento.

# para el caso adicional example4 caso adicional muestra el error cuando dos argumentos no son usados